### PR TITLE
Modvend 85 Flatten contact_person models, rename to 'contact'

### PR DESCRIPTION
**Pull request by komick**
## Purpose
In order to simplify the mod-vendor models, contact_person and vendor_contact_person models should be flattened into contact_person.  Also, to help distinguish models that will be separated into mod-contacts at a later date, the contact_person model is to be renamed 'contact', as contacts can refer to vendors as well as contacts within a vendor.

This change covers updates to models, ramls, APIs, module descriptors and permission sets, as well as sample vendor data and unit tests.

https://issues.folio.org/browse/MODVEND-85

## Approach
All example data and models were updated as part of changes to acq-models (https://github.com/folio-org/acq-models/pull/87).  All other changes have been incorporated into this PR.

#### TODOS and Open Questions
These changes will impact ui-vendors, and cannot be merged until into master until ui-vendors has been updated to reflect the changes in the vendor data structure (tracked in https://issues.folio.org/browse/UIV-76)

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [y] Code coverage on new code is 80% or greater
  - [y] Duplications on new code is 3% or less
  - [y] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [y] Were any API paths or methods changed, added or removed?
  - [y] Were there any schema changes?
  - [n] Did any of the interface versions change?
  - [y] Were permissions changed, added, or removed?
  - [n] Are there new interface dependencies?
  - [n] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [y] If not, please create them
  - [y] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [y] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [n] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [n] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/java/org/folio/rest/impl/ContactsAPI.java [hunk 1]
```diff
@@ -58,14 +58,14 @@ public void getVendorStorageContactPersons(String query, int offset, int limit,
           .setLimit(new Limit(limit))
           .setOffset(new Offset(offset));
 
-        PostgresClient.getInstance(vertxContext.owner(), tenantId).get(CONTACT_PERSON_TABLE, ContactPerson.class, fieldList, cql,
+        PostgresClient.getInstance(vertxContext.owner(), tenantId).get(CONTACT_PERSON_TABLE, Contact.class, fieldList, cql,
```
Comment (craigmcnally) [inline src/main/java/org/folio/rest/impl/ContactsAPI.java]: This code can be simplified via PgUtils... see https://github.com/folio-org/mod-orders-storage/pull/82 for details.  Whether you address this as part of this PR or not is up to you.  I plan on creating a separate story to make these changes across the rest of this module.

**Comment by craigmcnally** [comment]
John, in the pre-merge checklist, please leave the checkboxes as `[ ]` as opposed to entering in `y/n` this makes then interactive (clickable) like so:

* [ ] check or uncheck

**Review by craigmcnally — CHANGES_REQUESTED** [review]
Code coverage is low

**Comment by komick** [comment]
I've implemented PgUtils in the APIs and test coverage is now much higher.

**Review by craigmcnally — APPROVED** [review]

**Pull request merged**